webui/react/src/components/CreateGroupModal.tsx

```diff
@@ -193,7 +193,7 @@ const CreateGroupModalComponent: React.FC<Props> = ({ onClose, users, group }: P
           <Form.Item label={USERS_LABEL} name={USERS_NAME}>
             <Select mode="multiple" optionFilterProp="children" placeholder="Add Users" showSearch>
               {users?.map((u) => (
-                <Select.Option key={u.id} value={u.id}>
+                <Select.Option disabled={!u.isActive} key={u.id} value={u.id}>
```

Comment:
**question:** if you created a something like:
```
const activeUsers = (users ?? []).filter((user) => user.isActive);
```
then use the JSX:
```
{activeUsers.map((u) => { ... }}
```
Would you still run into the rendering issue you referred to? If the entry doesn't even exist for `activeUsers` then it should even show up as an antd.Select.Option...

Reply:
IIRC the issue that originally arose was when a user was in a group and later deactivated.